peps/pep-0728.rst

```diff
@@ -216,38 +235,36 @@ function parameters still apply::
 Inheritance
 -----------
 
-``__extra__`` is inherited the same way as a regular ``key: value_type``
-field. The same rules from :pep:`PEP 589 <589#inheritance>` apply.
-
-Additionally, we need to reinterpret the following rule to define how
-``__extra__`` interacts with it:
+The ``OtherFields[]`` annotated field (assuming its key is "__extra__") is inherited
+the same way as a regular ``key: value_type`` field. The same rules from
+:pep:`PEP 589 <589#inheritance>` apply.
 
     * Changing a field type of a parent TypedDict class in a subclass is not allowed.
 
 First, it is not allowed to change the type of ``__extra__`` in a subclass::
```

Comment:
If PEP 705 is accepted, `ReadOnly[]` on `OtherFields[]` will make it possible to refine the type in a subclass. However, I am unsure if this section is where the comment belongs to.